app/services/publisher_wallet_getter.rb

```diff
@@ -11,9 +11,16 @@ def initialize(publisher:)
   def perform
     return perform_offline if Rails.application.secrets[:api_eyeshade_offline]
     response = connection.get do |request|
-      # request.body = JSON.dump(params)
-      request.headers["Authorization"] = api_authorization_header
-      request.url("/v2/publishers/#{publisher.brave_publisher_id}/wallet")
+      if publisher.publication_type == :site
+        request.headers["Authorization"] = api_authorization_header
+        request.url("/v2/publishers/#{publisher.brave_publisher_id}/wallet")
+      elsif publisher.publication_type == :youtube_channel
+        request.headers["Authorization"] = api_authorization_header
+        request.url("/v1/owners/#{URI.escape(publisher.owner_identifier)}/wallet")
+      else
+        Rails.logger.warn("PublisherWalletGetter can't get wallet for publication_type #{publisher.publication_type.to_s}")
```

Comment:
This code path should never run, so let's also raise an exception and send it to Sentry (`begin; raise "bla"; rescue => e; Raven.capture_exception(e); end`)